### PR TITLE
Refactor/templates

### PR DESCRIPTION
**Pull request by naftalima**
## Objective and Motivation
- The flow wasn't good.

### CODE REVIEW

**Comment by naftalima** [comment]
I've already deployed and tested it: everything is working.

File: backend/battles/templates/battles/battle_detail.html [hunk 1]
```diff
@@ -1,11 +1,52 @@
...
-  <p>Creator: {{battle.creator}} </p>
-  <p>Opponent: {{battle.opponent}} </p>
+  <p>{{battle.creator}} challenged {{battle.opponent}} </p>
+
+
+  <h3 class="title">{{battle.creator}}'s team was:</h3>
+  <table>
+    <tr>
+      {% for pokemon in creator_pokemons %}
+      <td style="padding: 0 6px; width: fit-content;">
```
Comment (fjsj) [inline backend/battles/templates/battles/battle_detail.html:16]: You should avoid inline styles, fix the CSS instead.

**Review by fjsj — CHANGES_REQUESTED** [review]
Please check comments

File: backend/battles/templates/battles/battle_detail.html (hunk 1 above)
Comment (naftalima) [reply backend/battles/templates/battles/battle_detail.html:16]: are you talking about the style in base.html?

Comment (fjsj) [reply backend/battles/templates/battles/battle_detail.html:16]: You're using style attributes in HTML tags. Like here: `<td style="...">`
You should use CSS instead. Does that make sense?

Comment (naftalima) [reply backend/battles/templates/battles/battle_detail.html:16]: yes!!

**Review by fjsj — APPROVED** [review]
LGTM

**Pull request merged**